x-pack/plugins/security_solution/public/resolver/view/font_size.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/**
+ * Return a font-size based on a scale, minimum size, and a coefficient.
+ * @deprecated The projection matrix is already scaled non-linearly.
```

Comment:
:grey_question: I see this (marking things as deprecated) as a new pattern cropping up in comments. On one hand, it's nice to know but it's a little confusing without a suggestion of what to use in its place. Here it reads `The projection matrix is already scaled non-linearly` - which is a lot more detail than some of the other places I've seen this recently... but does that mean I should rewrite this function to correspond to the way the matrix is scaled? Or use a different function altogether? Some more detail about what the author is expected to do _instead_ would be helpful if we're going to continue marking deprecations (which I think we should).

Reply:
Also been thinking about this.  What we really need is a discussion. I added my thoughts on this function here: https://github.com/elastic/kibana/pull/77795#discussion_r490447640

I'm going to remove all '@deprecated' in the next PR unless i have any objections.